### DISCUSSION TITLE
How to add launch button?

### DISCUSSION THREAD

**Discussion by fortierq** [executablebooks/meta#773] [Q&A]
I would like to add a launch button to a notebook with jupyter-book, to open with [Basthon](https://basthon.fr/).  
However, I did not find much related to launch buttons, in the jupyter-book repo.  
Any hint on where the launch buttons are created?

**Comment by AakashGfude** [comment]
Hi @fortierq, details about launch buttons are on this docs page https://jupyterbook.org/en/stable/interactive/launchbuttons.html?highlight=launch%20buttons .  At present, we only support Binder, JupyterHub, and Google Colab.  
Adding a custom launcher like Bashton, is something we have implemented on a different theme called quantecon-book-theme, which also runs on jupyter-book. ex: https://python.quantecon.org/intro.html. Do you have something like this in your mind?

**Reply by jav-ed** [reply]
Adding additional buttons within the default book theme as shown in [https://python.quantecon.org/intro.html ](https://python.quantecon.org/intro.html) would be quite nice
